.github/workflows/yamllint.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: YAML lint workflow
+on:
+  pull_request:
+    paths:
```

Comment:
Should `.github/workflows/yamllint.yml` be a path here to match what we're doing for JSON?

Reply:
I think that's actually a typo from some of the testing I was doing out-of-band. We're not actually testing anything about the workflow.